statchatexpress/seeders/users.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "Username": "Jalen Washington",
+        "Email": "jswcodes@gmail.com",
+        "Password": "Diane1974"
+    }
```

Comment:
Someone suggested on workplace using chatgpt to generate mock/seed data. That would be pretty clever if we need more data down the line.

Reply:
yes i will make more data if necesary, for now i would like to keep the database to just one entry with dummy data and have users create their own profiles/stats later. if you think i should add more, i can do that